src/app/services/tasksService/tasks.service.ts

```diff
@@ -30,63 +30,63 @@ export class TasksService {
       type: 'General',
       completed: '80',
       dueDate: new Date('2019/11/17'),
-      id: this.itemId()
+      id: '1'
     },
     {
       title: 'The Flash Tutorial',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'General',
       completed: '70',
       dueDate: new Date('2019/12/29'),
-      id: this.itemId()
+      id: '2'
     },
     {
       title: 'Cleaning and Organising Your Computer',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'Clash',
       completed: '0',
       dueDate: new Date('2019/11/15'),
-      id: this.itemId()
+      id: '3'
     },
     {
       title: 'Android - UI Automation Test',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'General',
       completed: '80',
       dueDate: new Date('2019/11/17'),
-      id: this.itemId()
+      id: '4'
     },
     {
       title: 'The Flash Tutorial',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'General',
       completed: '70',
       dueDate: new Date('2019/11/16'),
-      id: this.itemId()
+      id: '5'
     },
     {
       title: 'Android - UI Automation Test',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'General',
       completed: '80',
       dueDate: new Date('2019/11/17'),
-      id: this.itemId()
+      id: '6'
     },
     {
       title: 'The Flash Tutorial',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'General',
       completed: '70',
       dueDate: new Date('2019/11/16'),
-      id: this.itemId()
+      id: '7'
     },
     {
       title: 'Cleaning and Organising Your Computer',
       projectName: 'CASD Wilson & Lamberton Middle Schools',
       type: 'Clash',
       completed: '0',
       dueDate: new Date('2020/01/05'),
-      id: this.itemId()
+      id: '8'
```

Comment:
**FIX:** This shouldn't have changed, we should still generate the unique id instead of hardcoded value.